openedx/core/djangoapps/discussions/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from .views import ConfigurationView
+
+
+urlpatterns = [
+    url(
+        r'^api/v0/(?P<course_key_string>.+)$',
```

Comment:
This gives us an initial URL of:
- http://127.0.0.1:18000/discussions/api/v0/course-v1:edX+DemoX+Demo_Course

Reply:
TODO: If we're actually going to support a version-able URL-structure (beneficial w/ breaking changes), we may want to adjust the file structure (`discussions/views/api/v0.py`, etc...).